test/python/test_ibmqjob_states.py

```diff
@@ -251,18 +258,39 @@ def test_only_final_states_cause_datailed_request(self):
 
                 with mock.patch.object(self._current_api, 'get_job',
                                        wraps=self._current_api.get_job):
-                    _ = job.status
+                    _ = job.status()
```

Comment:
You can remove the `_ =` part. It was needed because the linter did not understand that reading a member would cause a side effect so it forced me to do something with the value.